src/com/projectkorra/projectkorra/command/ToggleCommand.java

```diff
@@ -152,6 +169,7 @@ protected List<String> getTabCompletion(final CommandSender sender, final List<S
 			final List<String> elements = new ArrayList<String>();
 			for (final Element e : Element.getAllElements()) {
 				elements.add(e.getName());
+				elements.add(e.getName() + "Passives");
```

Comment:
What about toggling all passives at once?

Reply:
Sure. Will add that rn.